File: DKCpp/tests/dk_info.cpp
#include "dk_info.hpp"


//################################################################################
//# dk_info(message)
//#
//#include "dk_log.hpp"
//void dk_info(const char* message){
//	dk_log(message);
//}



//###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
#ifdef DKTEST
int main() {
	dk_info("test dk_info()");
    return 0;
}
#endif

File: DKCSharp/functions/dk_info.cs
public class APP{
        	
//############################################################################
//# dk_info(message)
//#
public static int dk_info(string message){
	APP.MAIN obj=new DKAPP.MAIN();
	obj.CompileAndRun("dk_log");
	//DKAPP.MAIN.myObj.CompileAndRun("dk_log");
	return 0;
}
    
	
	
	
	
//###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ###### DKTEST ######
public static void DKTEST(){
	dk_info("dk_info()");
}

}